packages/open-lens/package.json

```diff
@@ -195,15 +195,15 @@
     }
   },
   "dependencies": {
-    "@k8slens/application": "^6.5.0-alpha.0",
-    "@k8slens/feature-core": "^6.5.0-alpha.0",
-    "@k8slens/application-for-electron-main": "^6.5.0-alpha.0",
-    "@k8slens/core": "^6.5.0-alpha.0",
-    "@k8slens/ensure-binaries": "^6.5.0-alpha.0",
-    "@k8slens/generate-tray-icons": "^6.5.0-alpha.0",
-    "@k8slens/run-many": "^1.0.0",
-    "@k8slens/test-utils": "^1.0.0",
-    "@k8slens/utilities": "^1.0.0",
+    "@k8slens/application": "^6.5.0-alpha.1",
+    "@k8slens/application-for-electron-main": "^6.5.0-alpha.1",
+    "@k8slens/core": "^6.5.0-alpha.1",
+    "@k8slens/ensure-binaries": "^6.5.0-alpha.1",
+    "@k8slens/feature-core": "^6.5.0-alpha.1",
+    "@k8slens/generate-tray-icons": "^6.5.0-alpha.1",
+    "@k8slens/run-many": "^1.0.1-alpha.0",
+    "@k8slens/test-utils": "^1.0.1-alpha.0",
+    "@k8slens/utilities": "^1.0.1-alpha.0",
```

Comment:
are we actually changing these?

Reply:
Yes because they haven't actually been released yet